### PR TITLE
Cleaning `Sequence typehints`

### PR DESCRIPTION
**Pull request by kolibril13**
There were some old Union np.ndarray typehints, that are now updated to the new strategy.

### CODE REVIEW

File: manim/mobject/coordinate_systems.py [hunk 1]
```diff
@@ -194,8 +194,8 @@ class Axes(VGroup, CoordinateSystem):
 
     def __init__(
         self,
-        x_range=None,
-        y_range=None,
+        x_range: Optional[Sequence[float]] = None,
+        y_range: Optional[Sequence[float]] = None,
```
Comment (naveen521kk) [inline manim/mobject/coordinate_systems.py:198]: I don't `np.ndarray` comes under `Sequence`.

Comment (naveen521kk) [reply manim/mobject/coordinate_systems.py:198]: Can you confirm?

Comment (AntonBallmaier) [reply manim/mobject/coordinate_systems.py:198]: `np.ndarray` is a `Sequence` and Sequence is specific enough here.
https://docs.python.org/3/glossary.html#term-sequence

**Review by AntonBallmaier — APPROVED** [review]
lgtm

File: manim/mobject/coordinate_systems.py (hunk 1 above)
Comment (kolibril13) [reply manim/mobject/coordinate_systems.py:198]: I also agree with anton.
We also chose Sequence for all coordinates, which are np arrays:
https://manimce--1496.org.readthedocs.build/en/1496/contributing/documentation.html#adding-type-hints-to-functions-and-parameters

**Review by naveen521kk — APPROVED** [review]

**Pull request merged**